README.md

```diff
@@ -8,6 +8,62 @@ a threshold.
 The ultimately use case for this script is to run it periodically and use it to
 trigger alerts/notifications that the Product Delivery team can take heed of.
 
+## Architectural Overview
+
+### In Plain English
+
+This is stateless script, written in Python, meant to be executed roughly once a day.
+It uses `requests.get()` to query [Redash](https://sql.telemetry.mozilla.org) for
+a query's data that is pre-emptively run every 24 hours. The data it analyses is a
+list of every collection stored in Remote Settings (Kinto) along with a count of
+every possible status (e.g. `up_to_date`, `network_error`, etc.)
+The script sums the "good" statuses and compares it against the "bad" statuses and
+if that ratio (expressed as a percentage) is over a certain threshold it alerts
+by sending an event to Sentry which notifies the team by email.
+
+### Historical Strategy
+
+As of Feb 2019, Firefox clients that use Remote Settings only send Telemetry events
+in daily batches. I.e. it's _not_ real-time. The "uptake histogram" is buffered in
+the browser and will send periodically instead of as soon as possible. In Firefox
+Nightly (67 at the time of writing), we are switching to real-time Telemetry Events.
+
+On the Telemetry backend we're still consuming the older uptake histogram but once,
+what is Nightly today, becomes Release, we will switch the Redash query (where
```

Comment:
Will we switch when 67 becomes release? Maybe we'll have to wait a one or two more cycles to make sure our population becomes big enough?

Reply:
Days after 67 becomes release the numbers we currently get in https://sql.telemetry.mozilla.org/queries/61352 will quickly reduce to nothing, right?

Your patch to switch to Telemetry Events will *replace* the histogram stuff, no?

That reminds me, the change you landed in Nightly (which I can clearly see in `about:telemetry`) is already in effect. So, can I confirm that, with (a fork of) https://sql.telemetry.mozilla.org/queries/61352/source#158020, the numbers are all near 0 if I can filter that query down to just Nightlies?

Reply:
> Your patch to switch to Telemetry Events will _replace_ the histogram stuff, no?

Nope, it's both for now :)

Reply:
Wow! I totally missed that. Did the patch change after I first looked at it?
Either way, I'll make a note of this.